### PR TITLE
Window keylogger

### PR DESCRIPTION
**Pull request by deadex-ng**
@SANTHOSH17-DOT - I had issues with the previous keylogger [here](https://github.com/SANTHOSH17-DOT/attack-vectors-suite/issues/22). I'm on windows and the file can't run, windows automatically deletes the [main.py](https://github.com/SANTHOSH17-DOT/attack-vectors-suite/blob/main/KEYLOGGER/main.py). From the [readme](https://github.com/SANTHOSH17-DOT/attack-vectors-suite/tree/main/KEYLOGGER), I think it works only on Kali

I created another keylogger for windows.
Functionalities:
- It saves the log file.

Here is what the log file looks like:
```
Date - 2023-01-03 12:49:19.980322 | Timestamp - 1672742960
Window - Groove Music
Keys - 
j | 36 | 1672742938
l | 38 | 1672742938
k | 37 | 1672742938
f | 33 | 1672742939
j | 36 | 1672742939
l | 38 | 1672742939
j, l, k, f, j, l
```
The columns are key name, scan code, and time and the last line is a combination of all the key names.

I will add more functionality to key kelogger after this PR

### CODE REVIEW

**Comment by deadex-ng** [comment]
@SANTHOSH17-DOT  I have added a workflow to check Python code format using black.

**Comment by SANTHOSH17-DOT** [comment]
Looks great @deadex-ng. Could you create two subdirectories within keyloggers - one for windows and another for kali?
Let the kali version of the key logger be there. Also, Please squash all your commits into fewer commits

**Comment by deadex-ng** [comment]
@SANTHOSH17-DOT I'm not able to add the keylogger for Kali. Every time I open the project, the `main.py` is automatically deleted.

**Comment by SANTHOSH17-DOT** [comment]
Oh. I'll take a local copy of it and push it later. Just squash the commits. Rest looks fine

File: KEYLOGGER/windowsKeyLogger/Readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## This is a Windows keylogger
+
+## To use this keylogger:
+  * run `pip install requirements.txt`
+  * run `python main.py`
+  * You can now open any window on your pc and start typing.
+  * To stop the keylogger, type `ctrl+c` in the terminal
+  * The log file is located at `"c:/users/public/log.txt`
```
Comment (a-muk) [inline KEYLOGGER/windowsKeyLogger/Readme.md:8]: Hey! I'm one of the mentors for this project. Just wanted to ask, do you plan on adding a new log file or use the existing one? From the conversations, I believe that a new one has been created by you, but haven't been uploaded.

**Comment by SANTHOSH17-DOT** [comment]
No worries. Is your work safe?
Create a new branch from the main branch of your fork and copy your work there. Try this

**Comment by deadex-ng** [comment]
> No worries. Is your work safe? Create a new branch from the main branch of your fork and copy your work there. Try this @deadex-ng

@SANTHOSH17-DOT  My work is safe. Let me do what you've said.

File: KEYLOGGER/windowsKeyLogger/Readme.md (hunk 1 above)
Comment (deadex-ng) [reply KEYLOGGER/windowsKeyLogger/Readme.md]: > Hey! I'm one of the mentors for this project. Just wanted to ask, do you plan on adding a new log file or use the existing one? From the conversations, I believe that a new one has been created by you, but haven't been uploaded. Please clarify.

@a-muk - I made a new PR [here](https://github.com/SANTHOSH17-DOT/attack-vectors-suite/pull/24). I have also uploaded the log file that I created.

**Pull request closed without merging**